homeassistant/components/slide/cover.py

```diff
@@ -15,6 +15,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+CLOSED = "closed"
+CLOSING = "closing"
+OPENING = "opening"
```

Comment:
Could/Should this be a local enum instead?

Reply:
Something for later 👍